### DISCUSSION TITLE
Netdata functions: some work, some don't

### DISCUSSION THREAD

**Discussion by flipfloptech** [netdata/netdata#16678] [Q&A]
I recently just setup netdata on a storage solution that is giving me problems and it's pretty great in comparison to what shipped with the solution. But I noticed that some of the functions say you need to have a higher access level, does this just meaning a paid cloud account? or is there some other configuration that is missing? The documents don't really make it clear and the apps plugin seems to be working just fine.

**Comment by ilyam8** [comment]
Features that provide sensitive information and should not be publicly accessible require a connection to the Netdata Cloud:

- Processes
- Systemd-journal (Logs)
- Systemd-list-units
- Streaming
- Netdata-api-calls

No paid account required, free plan will do.

**Comment by flipfloptech** [comment]
Awesome, thank you!

**Reply by ilyam8** [reply]
Updated:
- docs in #16681
- messages in #16692
